src/components/DuosDatePicker.tsx

```diff
@@ -0,0 +1,197 @@
+              backgroundColor: '#216FB4',
+              '&:hover': {
+                backgroundColor: '#216FB4',
+              },
+              '&:focus': {
+                backgroundColor: '#216FB4',
+              },
+            },
+          },
+        },
+      },
+      MuiPickersYear: {
+        styleOverrides: {
+          yearButton: {
+            fontFamily: 'Montserrat',
+            color: '#000',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px 7px 20px',
+            '&.Mui-selected': {
+              backgroundColor: '#216FB4',
+              '&:hover': {
+                backgroundColor: '#216FB4',
+              },
+              '&:focus': {
+                backgroundColor: '#216FB4',
+              },
+            },
+          },
+        },
+      },
+      MuiPickersCalendarHeader: {
+        styleOverrides: {
+          root: {
+            fontSize:'2.5rem',
+          },
+          switchViewIcon:{
+            fontSize:'2.5rem',
+          },
+          label:{
+            fontFamily: 'Montserrat',
+            color: '#000',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px 7px 20px',
+          },
+        }
+      },
+    },
+  });
+
+  const CancelSelectActionBar = (props:PickersActionBarProps) => {
+    // Quirk of this control's usage pattern is the need to destructure the unused onSetToday and onClear from 'other'
+    // props.  This is in part because per mockup, this control does not support 'clear' or 'go to today' style buttons.
+    // eslint-disable-next-line no-unused-vars
+    const {onAccept, onCancel, onSetToday, onClear, actions, ...other} = props;
+    const buttons = actions?.map((actionType: React.Key | null | undefined) => {
+      switch (actionType) {
+        case 'cancel':
+          return (
+            <Button color={'secondary'} variant={'contained'} onClick={onCancel} key={actionType}>
+                            Cancel
+            </Button>
+          );
+
+        case 'accept':
+          return (
+            <Button color={'primary'} variant={'contained'} onClick={onAccept} key={actionType}>
+                            Select
+            </Button>
+          );
+
+        default:
+          return null;
+      }
+    });
+    return <DialogActions {...other}>{buttons}</DialogActions>;
+  };
+
+  const WeekendFormattedDay = (props: PickersDayProps<Dayjs>) => {
+    const isWeekendDay = props.day.day() === 0 || props.day.day() === 6;
+    const weekendStyle = isWeekendDay
+      ? {color: 'var(--weekend)',}
+      : {};
+    return <PickersDay {...props} sx={{...weekendStyle}} />;
+  };
+
+  return <ThemeProvider theme={theme}>
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DesktopDatePicker
+        closeOnSelect={false}
+        label={'Select a date'}
+        format={inputFormat}
+        value={value}
+        onAccept={(value)=>{onChange(value);}}
```

Comment:
Nit: I think you can just provide the method reference here?

Reply:
I'll tweak the types on both onChange and onError from `Function` to `any` and update these.